modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingWelcomePage.kt

```diff
@@ -164,6 +176,20 @@ private fun Content(
     }
 }
 
+@Composable
+private fun Confetti(
+    onConfettiShown: () -> Unit,
+) {
+    val lottieComposition by rememberLottieComposition(LottieCompositionSpec.RawRes(R.raw.confetti))
+    val progress by animateLottieCompositionAsState(lottieComposition)
+
+    LottieAnimation(
+        composition = lottieComposition,
+        contentScale = ContentScale.Crop,
+    )
+    if (progress == 1.0f) onConfettiShown()
```

Comment:
I really like how you made this a nice clean one-liner compared to something like

``` kotlin
if (progress == 1.0f) {
  onConfettiShown()
}
```

👍 👍